src/services/conversationsService.js

```diff
@@ -324,6 +324,21 @@ const changeReadOnlyState = async function(token, readOnly) {
 	}
 }
 
+const setConversationDescription = async function(token, description) {
+	try {
+		console.debug('setting conv desc')
+		const response = await axios.put(generateOcsUrl('apps/spreed/api/v3', 2) + `room/${token}/description`, {
+			description,
+		})
+		return response
+	} catch (error) {
+		console.debug('Error while setting conversation description', error)
```

Comment:
```suggestion
```

Reply:
logging the error can still be useful, we do it in many other functions